bin/extspice.py.orig

```diff
@@ -0,0 +1,63 @@
+<<<<<<< HEAD
+=======
+# extspice.py
+# Author:  Landon Burleson
+# Organization: Oklahoma State University
+# E-mail: landon.burleson@okstate.edu
+# Description: perform pex at terminal
+
```

Comment:
These .orig files are merge errors. You should probably remove them.

Reply:
+1, can you `git rm` those files (assuming those conflict have been resolved in the actual files)

Reply:
done..thx!